x-pack/qa/ml-native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/ScheduledEventsIT.java

```diff
@@ -257,6 +259,81 @@ public void testOnlineUpdate() throws Exception {
         assertEquals(0, buckets.get(8).getScheduledEvents().size());
     }
 
+    /**
+     * An open job that later gets added to a calendar, should take the scheduled events into account
+     */
+    public void testUpdatingOpenedJob() throws Exception {
+        TimeValue bucketSpan = TimeValue.timeValueMinutes(30);
+        String groupName = "opened-calendar-job-group";
+        Job.Builder job = createJob("scheduled-events-open-update", bucketSpan);
```

Comment:
Note that in all those integ-tests we try to name the job after the class/test itself. We found in the past that makes it easier to debug a failed test. Could you please adjust according to the final test name?

Reply:
definitely